project/src/main/scala/org/broadinstitute/clio/sbt/Dependencies.scala

```diff
@@ -51,7 +52,8 @@ object Dependencies {
     "io.circe" %% "circe-generic" % CirceVersion,
     "io.circe" %% "circe-generic-extras" % CirceVersion,
     "io.circe" %% "circe-parser" % CirceVersion,
-    "org.slf4j" % "slf4j-api" % Slf4jVersion
+    "org.slf4j" % "slf4j-api" % Slf4jVersion,
+    "com.fasterxml.uuid" % "java-uuid-generator" % UUIDVersion
```

Comment:
Same with these, can you move them to the right spot to keep the list alphabetical?